numpy/core/tests/test_indexing.py

```diff
@@ -282,8 +282,8 @@ def test_reversed_strides_result_allocation(self):
     def test_uncontiguous_subspace_assignment(self):
         # During development there was a bug activating a skip logic
         # based on ndim instead of size.
-        a = np.full((3, 4, 2), -1)
-        b = np.full((3, 4, 2), -1)
+        a = np.full((3, 4, 2), -1.)
```

Comment:
You are right, it was already changed after 1.11 apparently, so the changes are not necessary anymore.